proto/api/api.proto

```diff
@@ -223,11 +226,27 @@ message SuggestGasPriceResponse {
   uint64 gasPrice = 1;
 }
 
+// To be deprecated
 message EstimateGasForActionRequest {
+  iotextypes.Action action = 1;
+}
+
+message EstimateGasForActionsRequest {
   iotextypes.Execution execution = 1;
   string callerAddress = 2;
```

Comment:
remove this, and add it to EstimateGasForActionRequestV2. `callerAddress` should be applied to all types of actions